Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/Asset.tsx

```diff
@@ -125,6 +126,7 @@ function Asset(props: { AssetID: number }) {
         { Id: "additionalFields", Label: "Additional Fields" },
         { Id: "substations", Label: "Substations" },
         { Id: "meters", Label: "Meters" },
+        { Id: "eventChannels", Label: "Channels" },
```

Comment:
@gcsantos-gpa `Id` should probably be more descriptive since these are all Channels. Maybe `channels`